### ISSUE TITLE
Incorrect RIS fields

### ISSUE THREAD

**Issue by dsifford** [JabRef/jabref#3634]
Greetings.

I'm the developer of the Academic Blogger's Toolkit plugin for WordPress and some of my users were having difficulty importing RIS from jabref. Upon further investigation, it looks like you are currently mis-keying some fields. 

https://github.com/JabRef/jabref/blob/723af6dddfb4339e5f5fec3bb38869344201c287/src/main/java/org/jabref/logic/importer/fileformat/RisImporter.java#L147-L152

According to the RIS specification, `A1-A4` are exclusively _author_ fields. Only the `ED` field should be used for editors.

Related: https://github.com/dsifford/academic-bloggers-toolkit/issues/409#issuecomment-357453191

**Comment by dsifford** [comment]
Also, while I have your attention. It appears that your RIS exporter is leaving tons of blank lines (ostensibly where optional fields were skipped).

Here's an example that I just tried:

```txt
TY  - BOOK
AU  - Smith, Bob
AU  - Doe, Jan
AU  - Brown, Judy
A2  - Martin, Steve
A2  - Clark, Joe
T1  - Testing Book Title
PB  - Test Publisher

Y1  - 2015/october
VL  - 1












ER  -
```

**Comment by lenhard** [comment]
Thanks for reporting! This shouldn't be too hard to fix.

Related issue: #2607

**Comment by dsifford** [comment]
Sure thing. Happy to help.

Here's the list of field mappings that I use for RIS files: 

https://github.com/dsifford/astrocite/blob/668a9e4a0cb15a21a310d38e6e3f9ec5af7db9a0/packages/astrocite-ris/src/constants.ts#L62-L107

Also...

"stateful" fields: https://github.com/dsifford/astrocite/blob/668a9e4a0cb15a21a310d38e6e3f9ec5af7db9a0/packages/astrocite-ris/src/parser.ts#L58-L65

and finally, these:

https://github.com/dsifford/astrocite/blob/master/packages/astrocite-ris/src/parser.ts#L90-L109

The above fields include the standardized fields that you'd expect, but also non-standard (but highly used) fields that I've seen other applications export into (e.g., Zotero, Endnote, etc). 

Hope that's useful!

**Comment by lenhard** [comment]
The import-related problems described in this issue should be fixed by #3642. The export-related formatting issues are still present, but since this is about formatting and not functional correctness, I'll reclassify this issue from bug to enhancement.

**Comment by lenhard** [comment]
The blank lines in the export are now resolved as well by  #3661

So, I am closing this issue.
